# joint name
string joint_name

# change in velocity threshold
float64 velocity_delta_threshold

# change in torque threshold
float64 torque_delta_threshold

# change in velocity information

# previous joint velocity
float64 prev_joint_velocity
# timestamp of previous joint velocity (seconds)
float64 prev_joint_velocity_timestep

# current joint velocity
float64 curr_joint_velocity
# timestamp of current joint velocity (seconds)
float64 curr_joint_velocity_timestep

# change in torque information

# previous joint torque
float64 prev_joint_torque
# timestamp of previous joint torque (seconds)
float64 prev_joint_torque_timestep

# current joint torque
float64 curr_joint_torque
# timestamp of current joint torque (seconds)
float64 curr_joint_torque_timestep
